src/languages/java.js

```diff
@@ -179,6 +180,15 @@ export default function(hljs) {
           3: "title.class"
         }
       },
+      {
+        // Exceptions for hyphenated keywords
+        match: [
+          /non-sealed/
+        ],
+        className: {
+          1: "keyword"
+        }
+      },
```

Comment:
Lets use the simple form?
```suggestion
        match:  /non-sealed/,
        scope: "keyword"
      },
```

Reply:
Got it.